spark/processing/3.0/py3/docker/py37/Dockerfile.cpu

```diff
@@ -106,4 +106,8 @@ ENV SPARK_NO_DAEMONIZE TRUE
 
 WORKDIR $SPARK_HOME
 
+# Install the sagemaker feature store spark connector
+# https://docs.aws.amazon.com/sagemaker/latest/dg/batch-ingestion-spark-connector-setup.html
+RUN /usr/bin/python3 -m pip install sagemaker-feature-store-pyspark-3.0 --no-binary :all:
```

Comment:
Is this an officially released, publicly available lib, maintained by SageMaker and approved by AppSec to be used in the 1P the Spark Container